### ISSUE TITLE
🐛 Destination Clickhouse: normalization not working

### ISSUE THREAD

**Issue by Arkronus** [airbytehq/airbyte#9100]
## Environment
- **Airbyte version**:  0.35.1-alpha
- **OS Version / Instance**: Windows 10
- **Deployment**: docker-compose
- **Source Connector and version**: Postgres 0.4.0
- **Destination Connector and version**: Clickhouse 0.1.1
- **Severity**: Medium
- **Step where error happened**: Sync job 

## Current Behavior
I can transfer Raw Data, but I can`t use Basic normalization

## Expected Behavior
I should be able to use Basic normalization

## Logs

<details>
<summary>LOG</summary> 

```
2021-12-24 10:39:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Checking if airbyte/normalization-clickhouse:0.1.61 exists...
2021-12-24 10:39:52 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - airbyte/normalization-clickhouse:0.1.61 not found locally. Attempting to pull the image...
2021-12-24 10:39:54 [32mINFO[m i.a.c.i.LineGobbler(voidCall):82 - Image does not exist.
2021-12-24 10:39:54 [32mINFO[m i.a.w.t.TemporalAttemptExecution(lambda$getWorkerThread$2):177 - Completing future exceptionally...
io.airbyte.workers.WorkerException: Normalization Failed.
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:60) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:18) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at io.airbyte.workers.temporal.TemporalAttemptExecution.lambda$getWorkerThread$2(TemporalAttemptExecution.java:174) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: io.airbyte.workers.WorkerException: Could not find image: airbyte/normalization-clickhouse:0.1.61
	at io.airbyte.workers.process.DockerProcessFactory.create(DockerProcessFactory.java:97) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at io.airbyte.workers.normalization.DefaultNormalizationRunner.runProcess(DefaultNormalizationRunner.java:123) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at io.airbyte.workers.normalization.DefaultNormalizationRunner.normalize(DefaultNormalizationRunner.java:108) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	at io.airbyte.workers.DefaultNormalizationWorker.run(DefaultNormalizationWorker.java:55) ~[io.airbyte-airbyte-workers-0.35.1-alpha.jar:?]
	... 3 more
2021-12-24 10:39:54 [32mINFO[m i.a.w.t.TemporalAttemptExecution(get):144 - Stopping cancellation check scheduling...
2021-12-24 10:39:54 [33mWARN[m i.t.i.s.POJOActivityTaskHandler(activityFailureToResult):363 - Activity failure. ActivityId=ea5914e2-6cf6-3eac-9422-2531be5aebff, activityType=Normalize, attempt=1
java.util.concurrent.ExecutionException: io.airbyte.workers.WorkerException: Normalization Failed.

```

</details>

## Steps to Reproduce
1. Setup Clickhouse destination
2. Enable Basic Normalization
3. Expect error (see Logs)

**Comment by orayrrb** [comment]
Same request needed.

**Comment by alafanechere** [comment]
Hi @Arkronus, we have an open PR soon to be merged to enable basic normalization on clickouse: https://github.com/airbytehq/airbyte/pull/9072

**Comment by marcosmarxm** [comment]
This was solved by #9072
